### DISCUSSION TITLE
Is somebody working on a zustand expo dev-plugin, please?

### DISCUSSION THREAD

**Discussion by ericledonge-betonprovincial** [pmndrs/zustand#2473] [Ideas]
Expo has recently introduced a new way of debugging: https://github.com/expo/dev-plugins/tree/main/packages

I know I'd better contribute rather than just ask, but is somebody is currently working on this topic, please?

**Comment by gabimoncha** [comment]
I found this library created recently, but can't manage to make it work at the moment
https://github.com/jhonny1525/zustand-expo-devtools
